docs/journey.js

```diff
@@ -152,6 +153,12 @@ const calculateDailyBudget = () => {
     countryValid = true;
   });
 
+  $(document).keydown(function(e) {
```

Comment:
Why are we doing this?

Reply:
This is the key code the the "Enter" key. My guess is that it'll override the default behavior to submit the form.

Reply:
Yeah, I don't know why we are no longer submitting the form.

Reply:
This is so that on enter the form isn't submitted. This way when we use autocomplete on the Google Places textbox it doesn't show on error when people select an option and hit enter.